vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/patch_version.go

```diff
@@ -0,0 +1,34 @@
+package app
+
+var (
+	// OpenshiftMajor return openshift major version
+	OpenshiftMajor string
+	// OpenshiftMinor returns openshift minor version
+	OpenshiftMinor string
+	// OpenshiftGitVersion return git version of openshift
+	OpenshiftGitVersion string
+	// OpenshiftGitCommit returns git commit
+	OpenshiftGitCommit string
+	// OpenshiftBuildDate returns build date of openshift
+	OpenshiftBuildDate string
+)
+
+// OpenshiftVersionInfo stores version information about Openshift accessible to k8s packages
+type OpenshiftVersionInfo struct {
```

Comment:
What not use an `k8s.io/apimachinery/pkg/version.Info`?

Reply:
fixed